bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -467,6 +466,7 @@ def _dynamic_reconfigure_callback(self, config, level):
         else:
             rospy.loginfo('Debug images are disabled')
 
+        # TODO what is the meaning of this parameter? What is the fcnn output?
```

Comment:
It's the fcnn output publishing for the world model. It publishes the heatmap parts that are below the field boundary as a region of interest.